R/params.R

```diff
@@ -211,6 +213,10 @@ params_get_control <- function(param) {
   control
 }
 
+  params_allows_multiple <- function(param) {
```

Comment:
I don't think you need to factor out this function, but can just use `isTRUE(param$multiple)`.